driver/csiplugin/controllerserver.go

```diff
@@ -832,14 +832,14 @@ func (cs *ScaleControllerServer) checkMinScaleVersion(conn connectors.SpectrumSc
 }
 
 func (cs *ScaleControllerServer) checkSnapshotSupport(conn connectors.SpectrumScaleConnector) error {
-	/* Verify Spectrum Scale Version is not below 5.1.1-0 */
-	versionCheck, err := cs.checkMinScaleVersion(conn, "5110")
+	/* Verify Spectrum Scale Version is not below 5.1.2-1 */
+	versionCheck, err := cs.checkMinScaleVersion(conn, "5121")
 	if err != nil {
 		return err
 	}
 
 	if !versionCheck {
-		return status.Error(codes.FailedPrecondition, "the minimum required Spectrum Scale version for snapshot support with CSI is 5.1.1-0")
+		return status.Error(codes.FailedPrecondition, "the minimum required Spectrum Scale version for snapshot support with CSI is 5.1.2-1")
```

Comment:
We have to enhance this error message to "the minimum required Spectrum Scale version for CSI snapshot & cloning support is 5.1.1-0"